gtk3.20: Numix sidebar, Treeview progressbar and Notebook minimal fix.

megax opened the pull request:
Nemo:
![image](https://cloud.githubusercontent.com/assets/461493/14411229/38306072-ff42-11e5-94b2-ce4b1291fad6.png)

Progressbar:
![image](https://cloud.githubusercontent.com/assets/461493/14411224/1efd97a0-ff42-11e5-9baf-3d80cbbc65b8.png)

megax committed:
* Nemo sidebar treeview fix.

megax committed:
* Treeview progressbar numix style.

megax committed:
* Re-enabled.

megax commented:
@lots0logs Nemo, progressbar fix. :)

lots0logs commented:
@megax What exactly does this fix in Nemo? I can't tell from your screenshot :grimacing: I won't have time to review these until tonight...

megax commented:
NemoPlacesTreeView-disk-full settings. Old style:
![image](https://cloud.githubusercontent.com/assets/461493/14411583/d63a661e-ff4c-11e5-9ebe-c1f544cf08be.png)

The disk sider part of Nemo wasn't responding to changes so I replaced it so it can work and changed it's style to look better.

megax committed:
* Minimal notebook fix. Fixed 1px padding nemu desktop and other (gnome-terminal) application.

megax committed:
* Text warning fix.

megax commented:
I repaired in the Notebook, when selecting the tab, it jumped 1px. It's temporary, needs some change. The close button's size is fixed. There was a 1px padding on(vagy under nem tudom melyik lenne jó) Nemo desktop and gnome-terminal and on some other application, I fixed that too. :)

megax committed:
* Toolbar color fixed.

megax committed:
* Fixed flowbox name.

megax committed:
* switch padding fix.

megax commented:
Fixed switch padding:
Old:
![image](https://cloud.githubusercontent.com/assets/461493/14412823/611d4f1c-ff6b-11e5-8ade-d1939b5ba4fc.png)

New:
![image](https://cloud.githubusercontent.com/assets/461493/14412829/76173874-ff6b-11e5-967e-fa8b988fe2d0.png)

megax committed:
* .dim-label duplicated code. https://github.com/shimmerproject/Numix/blob/master/gtk-3.20/scss/widgets/_misc.scss#L5

lots0logs commented:
@megax I designed the new switch myself. Other than a few minor adjustments that I still need to do to it, I don't want to change it.

megax commented:
Ok, revert commit :)

megax committed:
Revert "* switch padding fix."

This reverts commit b2948a02e8f9f63c603245954e3f37785cc291e1.

megax commented:
Done patch :)

megax committed:
* notebook other background color fix.

megax committed:
* progressbar Transmission fix. Selection backdrop fix.

megax commented:
Minimal fix... :)

megax commented on gtk-3.20/scss/widgets/_notebook.scss:
```diff
@@ -7,22 +7,17 @@
-        border-style: solid;
-        border-color: border_normal($base_color);
-        border-radius: 0;
         background-color: $base_color;
```
Not optimal background color fix Nemo... Other application corrupted... (Rhythmbox)

lots0logs replied:
@megax So we need to revert this change right? I think it will be easier (for the both of us) going forward if you break changes up into separate pull requests. Something else that would be really helpful (though you don't have to do it if you don't want) is to include before and after screenshots of the changes for each pull request :grimacing:

megax replied:
@lots0logs Yes, background-color change revert (https://github.com/shimmerproject/Numix/commit/2eb312308438f6b51792854466406dd81bafe81a). Nemo fix later... :)
New pull requests separate ok :) and before, after screenshots :P

megax commented:
@lots0logs I close the pull request?

lots0logs commented:
Yes I got all the changes from this pull request. I will revert the background change in a few minutes. Thanks! :smile:
